src/interactions/panZoomInteraction.ts

```diff
@@ -1,4 +1,8 @@
 namespace Plottable.Interactions {
+
+  export type PanCallback = (e: Event) => void;
+  export type ZoomCallback = (e: Event) => void;
```

Comment:
Ideally it would be something like:
```
{
  xDomains: Map<Scale, X[]>;
  yDomains: Map<Scale, Y[]>;
}
```

but the typing on `PanZoom` doesn't allow for that since it's not generic on `<X, Y>` (also object-keyed `Map` isn't standard yet). For now we might want the callback to pass nothing (but still keep it type-defined), since the users can retrieve the domain information from the `Scale`s directly.